libraries/platforms/dromajo-vcs/bp_hb_platform.h

```diff
@@ -0,0 +1,96 @@
+// BlackParrot platform-specific header file
+// This file contains manycore-specific platform functions and constants
+
+#ifndef BP_HB_PLATFORM_H
+#define BP_HB_PLATFORM_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <bsg_manycore.h>
+#include <bsg_manycore_epa.h>
+
+// Host X and Y coordinates
+#ifndef HOST_X_COORD
+#error HOST_X_COORD is undefined
+#endif
+
+#ifndef HOST_Y_COORD
+#error HOST_Y_COORD is undefined
+#endif
+
+// BlackParrot coordinates
+// Note: Currently not all of these coordinates are in use
+// but might be useful during BlackParrot integration
+#ifndef BP_COORDS
+#define BP_POD_X 0
+#define BP_POD_Y 1
+// Coordinates of BP-HB bridge used for manycore communication
+#define BP_HOST_LINK_X 15
+#define BP_HOST_LINK_Y 9
+// Coordinates of BP-HB bridge used for DRAM loads and stores
+#define BP_DRAM_0_LINK_X 15
+#define BP_DRAM_0_LINK_Y 10
+#define BP_DRAM_1_LINK_X 15
+#define BP_DRAM_1_LINK_Y 11
+#endif
+
+// Memory mapped addresses to interact with the BlackParrot host (aka simulator)
+#define HB_MC_HOST_EPA_ARGS_START 0x0000
+#define HB_MC_HOST_EPA_ARGS_FINISH 0x00FF
+#define HB_MC_HOST_EPA_CONFIG_START 0x0100
+#define HB_MC_HOST_EPA_CONFIG_FINISH 0x01FF
+#define HB_MC_HOST_EPA_RESET_DONE 0x0200
+#define HB_MC_HOST_EPA_TX_VACANT 0x0300
+
+#define HB_MC_HOST_OP_FINISH_CODE 0xFFFFFFFF
```

Comment:
On second thought, is this HB_MC or HB_BP?

Reply:
I kept these HB_MC because the EPAs in `bsg_manycore_epa.h` was defined with HB_MC_*.

Reply:
But I think HB_BP makes more sense here.